backend/typescript/services/implementations/authService.ts

```diff
@@ -10,6 +10,8 @@ import getErrorMessage from "../../utilities/errorMessageUtil";
 
 const Logger = logger(__filename);
 
+const ADMIN_EMAIL = "communityfridge@uwblueprint.org";
```

Comment:
I don't think we need a variable here, they will always be reaching out to this address so let's just have it hardcoded in the email